packaging/technical-addon/Makefile

```diff
@@ -1,12 +1,19 @@
 # Used for building
-SOURCE_DIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-BUILD_DIR=$(realpath $(SOURCE_DIR)/../../)/build
-COMMON_MAKEFILE:=$(abspath $(SOURCE_DIR)/../../Makefile.Common)
-
+THIS_MAKEFILE_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
```

Comment:
There's a lot of path logic here, is there some reason we can't just use relative paths and `include` from there? (In other words, is there some reason we can't just do `include ../../Makefile.Common` and `include ../../instrumentation/Makefile`?)